app/src/main/java/com/architect/coders/mu8/Utils.kt

```diff
@@ -0,0 +1,11 @@
+package com.architect.coders.mu8
+
+const val TS = BuildConfig.TS
```

Comment:
Rename `ts` for `TIMESTAMP`, avoid the usage of short name variables that doesn't tell exactly what it is in an obvious way.